ubuntu.22.04/spec/ubuntu.22.04.tests.ps1

```diff
@@ -82,7 +82,8 @@ Describe  'installed dependencies' {
     }
     
     It 'has gke-gcloud-auth-plugin installed' {
-        gke-gcloud-auth-plugin --version | Select -First 1 | Should -be 'Kubernetes v1.28.2-alpha+58ec6ae34b7dcd9699b37986ccb12b3bbac88f00'
+        #We use belike here as the hash after the 'alpha+' changes and isn't that important
+        gke-gcloud-auth-plugin --version | Select -First 1 | Should -beLike 'Kubernetes v1.28.2-alpha+*'
```

Comment:
The hash at the end changed and isn't really important to test if the hash is exactly the same